draft-ietf-add-ddr.md

```diff
@@ -303,9 +300,10 @@ Often, the various supported encrypted DNS protocols will be accessible using
 the same hostname. In the example above, both DoH and DoT use the name
 `resolver.example.com` for their TLS certificates. If a deployment uses a
 different hostname for one protocol, but still wants clients to treat both DNS
-servers as designated, the TLS certificates MUST include both names in the
-SubjectAlternativeName fields. Note that this name verification is not related
-to the DNS resolver that provided the SVCB answer.
+servers as designated, the TLS certificates MUST include the name for which
+the client issued the query in the SubjectAlternativeName field. Note that
```

Comment:
I think that you will need to be more precise.  There are lots of DNS queries involved here.

Reply:
Okay, I restructured this paragraph to be clearer.